### PR TITLE
MAISTRA-2367: Add SME converter webhook

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -54,6 +54,19 @@ replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.3
...
+
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.3
+
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.18.3
+
+replace k8s.io/code-generator => k8s.io/code-generator v0.18.3
+
+replace github.com/go-logr/zapr => github.com/go-logr/zapr v0.2.0
+
+// FIXME: Waiting on https://github.com/maistra/api/pull/7 to be merged
```
Comment (jwendell) [inline go.mod]: https://github.com/maistra/api/pull/7

**Comment by rcernich** [comment]
This looks good to me.  I think we should consider moving the document generation out of istio-operator and into api (in the future).

**Review by rcernich — APPROVED** [review]

**Comment by dgn** [comment]
> This looks good to me. I think we should consider moving the document generation out of istio-operator and into api (in the future).

I agree. Should we file an issue for that work?

**Review by dgn — APPROVED** [review]

**Comment by jwendell** [comment]
> > This looks good to me. I think we should consider moving the document generation out of istio-operator and into api (in the future).
> 
> I agree. Should we file an issue for that work?

Done: https://issues.redhat.com/browse/MAISTRA-2399

**Pull request merged**